src/main/java/org/fountainmc/api/command/CommandBuilder.java

```diff
@@ -0,0 +1,101 @@
+package org.fountainmc.api.command;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class CommandBuilder {
```

Comment:
A builder to build an interface is redundant when you can create one class that can do the same thing.